lib/blpapi-wrapper.ts

```diff
@@ -99,7 +116,7 @@ function subscriptionsToServices(subscriptions: Subscription[]): string[] {
         // XXX: note that we shoud probably capture what the default service is to use when
         //      reading in the session options.  However, when not specified, it is
         //      '//blp/mktdata'.
-        return match ? match[0] : '//blp/mktdata';
+        return (subscription.serviceUri = match ? match[0] : '//blp/mktdata');
```

Comment:
If not for seeing this in the diff, I would likely miss the assignment. Maybe break this into two statements for clarity.